site/en/docs/devtools/blog/index.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: "layouts/doc-post.njk"
+title: "DevTools Engineering Blog"
+description: "The Chrome DevTools engineering blog - by the engineers who build the DevTools."
+---
+
+Below is a list of everything that's been covered in the *Chrome DevTools Engineering Blog* series.
+
+* [CSS-in-JS support in DevTools](/blog/devtools-engineering-10)
```

Comment:
Was it a conscious choice to use non-descriptive URLs like `/blog/devtools-engineering-{1..9999}`? If possible, I’d much prefer to keep the more descriptive file names and URLs.

Reply:
Changed URLs to `/blog/devtools/descriptive-name`

Reply:
Changed generic URLs without `devtools` subdir. E.g. `/blog/css-in-js`.